CesiumGeometry/include/CesiumGeometry/AxisAlignedBox.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
 
-#include <CesiumGeometry/Library.h>
+#include "Library.h"
```

Comment:
We used to do this for headers in the same directory, but lately we've just been using the qualified name in angle brackets for consistency.

Reply:
Okay! I thought that went against our style guide, but I'll revert it.

Reply:
It might be worth adding a section about #include style to the style guide? Especially since this is a change from [SF.12](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf12-prefer-the-quoted-form-of-include-for-files-relative-to-the-including-file-and-the-angle-bracket-form-everywhere-else).

Reply:
I reopened https://github.com/CesiumGS/cesium-native/issues/991 noting this change.

Reply:
That might be fixable with `.clang-format`

Reply:
Hm maybe not: https://github.com/llvm/llvm-project/issues/39735